utils/impl/IsOmitted.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2022 Extremely Heavy Industries Inc.
+ */
+import { PlainObject } from '@xh/hoist/core/types/Types';
+import {isFunction} from 'lodash';
+
+/**
+ * @internal
+ */
+export function isOmitted(obj: PlainObject): boolean {
```

Comment:
Should the parameter type be more specific?  Also looks like the implementation allows for `obj` to be `null`, in which case I think this would return `undefined` instead of a `boolean `.

Would this be clearer?:

```
export function isOmitted(obj?: {omit?: Thunkable<boolean>}): boolean {
    return isFunction(obj?.omit) ? obj.omit() : !!obj?.omit;
}
```